deploy/bundle/manifests/devworkspace-operator.clusterserviceversion.yaml

```diff
@@ -304,7 +304,7 @@ spec:
                 - name: RELATED_IMAGE_web_terminal_tooling
                   value: quay.io/wto/web-terminal-tooling:latest
                 - name: RELATED_IMAGE_pvc_cleanup_job
-                  value: quay.io/libpod/busybox:1.30.1
+                  value: registry.access.redhat.com/ubi8/ubi-minimal:8.4
```

Comment:
what about using it in other deployment files https://github.com/devfile/devworkspace-operator/blob/main/deploy/templates/components/manager/manager.yaml#L92 ?

Reply:
Done